integrated_channels/blackboard/client.py

```diff
@@ -175,6 +224,21 @@ def _validate_course_id(course_id, external_id):
                 HTTPStatus.NOT_FOUND.value
             )
 
+    @staticmethod
+    def _validate_full_course_metadata(channel_metadata_items):
```

Comment:
A nit but isn't this one item, `channel_metadata_item` ?

Reply:
yes, but the metadata is broken into three separate metadata objects

Reply:
So this is checks for the presence of `course_metadata`, 'course_content_metadata`, and `course_child_content_metadata`

Reply:
that's cool, but sorry I was not clear. I thought that variable name shoiuld be `channel_metadata_item` since we work with one item at a time, no?

Reply:
🤦  You know, I think I originally read this parameter as `"content_metadata_items"` and as such thought "yes, there are mutliple content items here". Okay, I'm fine with making this singular (or changing to `content_metadata_items`